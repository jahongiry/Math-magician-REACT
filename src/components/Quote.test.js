import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

it('renders if there is Quote', () => {
  const three = renderer.create(<Quote />).toJSON();
  expect(three).toMatchSnapshot();
});
