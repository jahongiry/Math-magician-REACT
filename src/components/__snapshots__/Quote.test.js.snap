// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders if there is Quote 1`] = `
<div
  className="quotecontainer"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
